bleak/backends/bluezdbus/agent.py

```diff
@@ -0,0 +1,210 @@
+"""
+Agent
+-----
+
+This module contains types associated with the BlueZ D-Bus `agent api
+<https://github.com/bluez/bluez/blob/master/doc/agent-api.txt>`.
+"""
+
```

Comment:
```suggestion

import sys
from typing import TYPE_CHECKING

from bleak.args.bluez import BlueZNotifyArgs

if TYPE_CHECKING:
    if sys.platform != "linux":
        assert False, "This backend is only available on Linux"

```

Needed to avoid type checking failures on other platforms.